qiskit/basics/getting_started_with_qiskit_terra.ipynb

```diff
@@ -121,7 +121,7 @@
    "source": [
     "## Visualize Circuit\n",
     "\n",
-    "You can visualize your circuit using Qiskit Terra `circuit_drawer`, which plots circuit in the form found in many textbooks."
+    "You can visualize your circuit using Qiskit Terra `circuit_drawer`, which plots circuit in the form found in many textbooks. For more details on the options when visualizing a circuit see: [Visualizing a Quantum Circuit](visualizing_a_quantum_circuit.ipynb)"
```

Comment:
I think this link when we make it html will be broken i would just remove

Reply:
Done